cextern/erfa/erfaextra.h

```diff
@@ -0,0 +1,59 @@
+/*
+** Copyright (C) 2016-2017, NumFOCUS Foundation. 
+**
+** Licensed under a 3-clause BSD style license - see LICENSE
+**
+** This file is NOT derived from SOFA sources
+*/
+
+
+#ifndef _ERFA_EXTRA_H
+#define _ERFA_EXTRA_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/* 
+** Returns the package version
+** as defined in configure.ac
+** in string format
+*/
+const char* eraVersion();
+
+/* 
+** Returns the package major version
+** as defined in configure.ac
+** as integer
+*/
+int eraVersionMajor();
+
+/* 
+** Returns the package minor version
+** as defined in configure.ac
+** as integer
+*/
+int eraVersionMinor();
+
+/* 
+** Returns the package micro version
+** as defined in configure.ac
+** as integer
+*/
+int eraVersionMicro();
+
+/* 
+** Returns the orresponding SOFA version
+** as defined in configure.ac
+** in string format
+*/
+const char* eraSofaVersion();
```

Comment:
@eteq - These declarations are causing many warnings:
```
    cextern/erfa/erfaextra.h:23:1: warning: function declaration isn’t a prototype [-Wstrict-prototypes]                                                                                       
     const char* eraVersion();                                                                                                                                                                 
     ^                                                                                                                                                                                             cextern/erfa/erfaextra.h:30:1: warning: function declaration isn’t a prototype [-Wstrict-prototypes]                                                                                       
     int eraVersionMajor();                                                                                                                                                                         ^                                                                                                                                                                                         
    cextern/erfa/erfaextra.h:37:1: warning: function declaration isn’t a prototype [-Wstrict-prototypes]                                                                                       
     int eraVersionMinor();                                                                                                                                                                    
     ^                                                                                                                                                                     
```

I guess it should use `(void)` instead of `()` : https://stackoverflow.com/questions/42125/function-declaration-isnt-a-prototype